src/researchhub/settings.py

```diff
@@ -555,19 +555,13 @@ def silky_capture(request):
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
 EMAIL_WHITELIST = [
-    "leosun@quantfive.org",
-    "patrick@quantfive.org",
-    "patrick.lu@berkeley.edu",
     "pdj7@georgetown.edu",
     "bank@researchhub.com",
     "kobeattias@gmail.com",
     "kobe@researchhub.com",
     "kobe+1@researchhub.com",
     "contact@notesalong.com",
```

Comment:
Should we remove this one too?

Reply:
I believe this is @yattias, not sure if it's still used?

I suggest to eventually move all of this into configuration, but maybe in a next step.